applications/luci-app-mwan3/luasrc/model/cbi/mwan/ruleconfig.lua

```diff
@@ -15,62 +14,72 @@ local ipsets = util.split(util.trim(util.exec("ipset -n -L 2>/dev/null | grep -v
 m = Map("mwan3", translatef("MWAN Rule Configuration - %s", arg[1]))
 m.redirect = dsp.build_url("admin", "network", "mwan", "rule")
 
-mwan_rule = m:section(NamedSection, arg[1], "rule", "")
-mwan_rule.addremove = false
-mwan_rule.dynamic = false
+s = m:section(NamedSection, arg[1], "rule", "")
+s.addremove = false
+s.dynamic = false
 
-src_ip = mwan_rule:option(Value, "src_ip", translate("Source address"),
+o = s:option(ListValue, "family", translate("Internet Protocol"))
+o.default = ""
+o:value("", translate("IPv4 and IPv6"))
+o:value("ipv4", translate("only IPv4"))
+o:value("ipv6", translate("only IPv6"))
```

Comment:
could we just change the `only` prefix?
I notice the same code in firewall.js
```
        o.value('', _('IPv4 and IPv6'));
        o.value('ipv4', _('IPv4 only'));
        o.value('ipv6', _('IPv6 only'));
```

Reply:
Done with commit https://github.com/openwrt/luci/commit/1c482dde3d33f79fd7c4e8f131f210b60bc4223c